index.html

```diff
@@ -0,0 +1,263 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav class="navbar">
+            <a href="">
+                <h2>My Logo</h2>
+            </a>
+            <img src="images/hamburger.svg" alt="menu button">
+        </nav>
+    </header>
+    <section id="headline">
+        <h1>I’m Thinus Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="social-links">
+            <li>
+                <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/linkedinicon.svg" alt="LinkedIn"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector.svg" alt="Medium"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/github.svg" alt="Github"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector (1).svg" alt="Angel List"></a>
+            </li>
+        </ul>
+    </section>
+
+    <section id="works">
+        <div id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio.svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <h4 id="project_title">Tonic</h4>
+                    <section id="frame2">
+                        <section id="client">
+                            <h2 id="pcanopy">CANOPY</h2>
+                        </section>
+                        <img id="counter">
+                        <section id="role">
+                            <h3 id="pcareer">Back End Dev</h3>
+                        </section>
+                        <img id="counter">
+                        <section id="year_label">
+                            <h5 id="pcareer">2015</h5>
+                        </section>
+                    </section>
+                </section>
+                <p id="sub_text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <section id="tags">
+                    <section id="tag1">
+                        <p id="thtml">html</p>
+                    </section>
+                    <section id="tag2">
+                        <p id="tcss">css</p>
+                    </section>
+                    <section id="tag3">
+                        <p id="tjavascript">javascript</p>
+                    </section>
+                </section>
```

Comment:
- [ ] According to best practices, you can only use `section` tag to group related elements. The format below might be helpful for you.

<section>
                <div>
                    <img src="resources/img/SnapshootPortfolio.png" alt="snapshoot">
                </div>
                <div>
                    <div>
                        <h1 class="tonic">Tonic</h1>
                        <ul>
                            <li><a href="#">CANOPY</a></li>
                            <li><a href="#">Back End Dev</a></li>
                            <li><a href="#">2015</a></li>
                        </ul>
                    </div>
                    <p>Proof read my works here, no account sign-ups required</p>
                    <ul>
                        <li>html</li>
                        <li>css</li>
                        <li>javascript</li>
                    </ul>